torch/_dynamo/variables/misc.py

```diff
@@ -868,8 +877,21 @@ def call_function(
                 msg += f"confg.use_numpy_random_stream={config.use_numpy_random_stream}"
                 unimplemented(msg)
 
-            # TODO(larryliu0820): currently assuming all numpy.* functions are returning a ndarray that can be
-            #  wrapped by NumpyNdarrayVariable which is wrong!
+            constant_args = check_constant_args(args, kwargs)
+            unspec_python_args = check_unspec_python_args(args, kwargs)
+
+            if self.can_constant_fold_through(func) and (
+                constant_args or unspec_python_args
+            ):
+                # constant fold
+                return variables.ConstantVariable.create(
+                    self.as_python_constant()(
+                        *[x.as_python_constant() for x in args],
```

Comment:
```suggestion
                        *(x.as_python_constant() for x in args),
```

Reply:
I copied this from the equivalent TorchVariable code. We should add linter rules if we want these things to be consistent in the codebase